keystore_test.go

```diff
@@ -127,3 +127,46 @@ func TestGetKey(t *testing.T) {
 
 	fmt.Printf("%+v\n", pk2.GetPublic())
 }
+
+func TestKeystoreManager_ExportKey(t *testing.T) {
```

Comment:
this would be a good candidate for a table-driven test, as we might want to check edge cases for key export and add regression cases easily if any issues come up